src/components/Centered/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const Centered = ({ flex, children }) => (
+  <div className="centered">
+    {children}
+    <style jsx>{`
+      .centered {
+        ${flex
```

Comment:
Generally you can use classNames to better express many css properties, i.e. [`className={classNames('centered', flex && 'flex')}`](https://www.npmjs.com/package/classnames#usage)
Then you can simply
```
.centered {
  text-align:center;
}

.flex {
...
}
```

Reply:
Fixed.